PKGBUILD

```diff
@@ -3,32 +3,50 @@
 
 pkgname=linx-server
 pkgver=2.3.5
-pkgrel=1
-pkgdesc='Self-hosted file/code/media sharing website '
-arch=('i686' 'x86_64' 'arm')
+pkgrel=2
+pkgdesc='Self-hosted file/code/media sharing website'
+arch=('x86_64')
 url='https://github.com/andreimarcu/linx-server'
 license=('GPL3')
-install=linx-server.install
```

Comment:
I agree that using sysusers/tmpfiles is better than creating the user/group/dir manually, but you still have to call `systemd-sysusers/tmpfiles` somewhere.

Reply:
In `package()` those file copied to `/usr/lib/{sysusers.d,tmpfiles.d}` and this triggers pacman hooks from `/usr/share/libalpm/hooks/{20-systemd-sysusers.hook,30-systemd-tmpfiles.hook}`.

Reply:
Oh, I didn't know about those hooks, nvm then.